certbot/plugins/common.py

```diff
@@ -262,15 +263,49 @@ def _setup_challenge_cert(self, achall, cert_key=None):
         return response
 
 
-# test utils used by certbot_apache/certbot_nginx (hence
+# utils used by certbot_apache/certbot_nginx (hence
 # "pragma: no cover") TODO: this might quickly lead to dead code (also
 # c.f. #383)
 
-def setup_ssl_options(config_dir, src, dest):  # pragma: no cover
-    """Move the ssl_options into position and return the path."""
-    option_path = os.path.join(config_dir, dest)
-    shutil.copyfile(src, option_path)
-    return option_path
+def install_ssl_options_conf(options_ssl, options_ssl_digest, mod_ssl_conf_src,
+    all_ssl_options_hashes, logger): # pragma: no cover
+    """Copy Certbot's SSL options file into the system's config dir if required."""
```

Comment:
Since this module is [documented publicly](https://certbot.eff.org/docs/api/plugins/common.html) and other plugin authors may use it, it'd be good to document the parameters of this function.